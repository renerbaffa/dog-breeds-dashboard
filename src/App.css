/* tablet */
.wrapper {
  width: 100%;
  height: 100%;
  overflow: hidden;
  position: absolute;
  top: 0;
  left: 0;
  background-color: #f0f0f0;
  font-family: "Segoe UI";
}

.app {
  border-radius: 3px;
  box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.06), 0 2px 5px 0 rgba(0, 0, 0, 0.2);
  margin: 0 auto;
  background-color: #f7f7f7;
  display: flex;
  overflow: hidden;
  position: relative;

  top: 0;
  width: 100%;
  height: 100%;
}

.container {
  display: flex;
  height: 100%;
  overflow: hidden;
  position: absolute;
  top: 0;
  width: 100%;
}

.left-panel {
  width: 30%;
  position: relative;
  border-right: 1px solid lightgray;
}

.right-panel {
  width: 70%;
  overflow: hidden;
  position: relative;
}




/* web */
@media screen and (min-width: 1441px) {
  .app {
    height: calc(100% - 38px);
    top: 19px;
    width: 1396px;
  }
}

/* tablet */
@media screen and (max-width: 1300px) and (min-width: 1025px) {
  .left-panel {
    width: 35%;
  }

  .right-panel {
    width: 65%;
  }
}

/* mobile */
@media screen and (max-width: 648px) {
  .app-wrapper {
    overflow-x: auto;
  }

  .app {
    min-width: 648px;
  }
}

@media screen and (max-width: 900px) {
  .left-panel {
    width: 40%;
  }

  .right-panel {
    width: 60%;
  }
}
