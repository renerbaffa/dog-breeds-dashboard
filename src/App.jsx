import React from 'react';

import styles from './App.css';

const App = () => (
  <div className={styles.wrapper}>
    <div className={styles.app}>
      <div className={styles.container}>
        <div className={styles.leftPanel} />
        <div className={styles.rightPanel} />
      </div>
    </div>
  </div>
);

export default App;
