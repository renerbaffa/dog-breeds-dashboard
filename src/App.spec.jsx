import React from 'react';
// import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import App from './App';

describe('<App />', () => {
  it('should render', () => {
    const renderedValue = renderer.create(<App />).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
