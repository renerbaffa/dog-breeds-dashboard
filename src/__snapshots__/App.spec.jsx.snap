// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render 1`] = `
<div
  className="wrapper"
>
  <div
    className="app"
  >
    <div
      className="container"
    >
      <div
        className="leftPanel"
      />
      <div
        className="rightPanel"
      />
    </div>
  </div>
</div>
`;
